presto-server-rpm/pom.xml

```diff
@@ -230,6 +230,30 @@
                     </packages>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>enforce-file-size</id>
+                        <phase>verify</phase>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <requireFilesSize>
+                                    <maxsize>1073741824</maxsize>
```

Comment:
```
<!-- see https://github.com/prestosql/presto/issues/3401 for explanation -->
```

(same below)

Reply:
> eg if you lower this number to 800M, does the build fail?
Yes it does fail

Reply:
per above, please add the issue link in the comment
```suggestion
                                    <!-- see https://github.com/prestosql/presto/issues/3401 for explanation -->
                                    <maxsize>1073741824</maxsize>
```

Reply:
I'd rather have the explanation inline. It's a simple explanation and doesn't require linking to an issue. Having things self contained when possible is best.

Reply:
We should probably use `1073741823` since it's not clear if 1GB is at or above the limit.